unish.sut.php

```diff
@@ -75,5 +79,11 @@ function unish_setup_sut($unish_sandbox) {
     symlink("$target_dir/vendor", __DIR__ . '/vendor');
   }
 
+  // Create a symlink to modules directory in Drush root, so that PHPUnit is able to discover module Unish tests.
```

Comment:
This symlink is a bit unfortunate IMO. For one thing, it looks like it will add a code to PHPStorm project when working on Drush. Maybe we can avoid it?

Reply:
Well... I have no idea how can I remove it because this line in `phpunit.xml.dist`:

```xml
<directory>../modules/unish/*/tests/src/Unish</directory>
```

Another option would have been to add a symlink directly in the SUT install as `/path/to/sut/drush-sut/vendor/drush/drush` but the problem is that `/path/to/sut/drush-sut/vendor/drush/drush` is a symlink and it's the same.

However, I see a big advantage to bring the modules code in PhpStorm. As I already told in other comment, you're able to design Drush commands in contrib module from the Drush project. 

I will think more on this but I don't see an alternative for discovering the tests from contrib. I don't think that `phpunit.xml.dist` knows to use/read env vars so that we could pass the SUT dir (?)